spec/features/bank_account_transactions_spec.rb

```diff
@@ -31,15 +31,10 @@
       end
     end
 
-    it "displays selected account's transactions " do
+    it "displays selected account's transactions and not display not own transactions" do
```

Comment:
@dissident instead you may just write `displays only selected account's transactions`